bgpd/bgp_routemap.c

```diff
@@ -379,13 +379,20 @@ route_match_script(void *rule, const struct prefix *prefix, void *object)
 			      status_match = LUA_RM_MATCH,
 			      status_match_and_change = LUA_RM_MATCH_AND_CHANGE;
 
+	struct attr newattr;
+
+	newattr.med = path->attr->med;
+	newattr.nh_ifindex = path->attr->nh_ifindex;
+	newattr.aspath = path->attr->aspath;
+	newattr.local_pref = path->attr->local_pref;
```

Comment:
The next API update with an updated `frrscript_get_result` will prevent us from having to do this manual copy and pasting to get new results